<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KinectViewer</class>
 <widget class="QMainWindow" name="KinectViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>499</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KinectViewer</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>10</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStopStream">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>436</y>
      <width>121</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>Stream running</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetCamera">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>376</y>
      <width>121</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom to fit</string>
    </property>
   </widget>
   <widget class="QSlider" name="sldClipDistance">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>71</y>
      <width>65</width>
      <height>277</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3000</number>
    </property>
    <property name="singleStep">
     <number>50</number>
    </property>
    <property name="sliderPosition">
     <number>3000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblClipDistance">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>8</y>
      <width>85</width>
      <height>53</height>
     </rect>
    </property>
    <property name="text">
     <string>Clip far data [mm]</string>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidgetRegistration" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddCloud">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>316</y>
      <width>81</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Add cloud</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoveCloud">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>352</y>
      <width>81</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove clouds</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="boxMaxIterations">
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>344</y>
      <width>57</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>150.000000000000000</double>
    </property>
    <property name="value">
     <double>25.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistration">
    <property name="geometry">
     <rect>
      <x>848</x>
      <y>316</y>
      <width>81</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>728</x>
      <y>324</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max iterations</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtTransformation">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>404</y>
      <width>177</width>
      <height>89</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>384</y>
      <width>185</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Homogeneous transformation matrix</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtAngleX">
    <property name="geometry">
     <rect>
      <x>724</x>
      <y>404</y>
      <width>73</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtAngleY">
    <property name="geometry">
     <rect>
      <x>724</x>
      <y>436</y>
      <width>73</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtAngleZ">
    <property name="geometry">
     <rect>
      <x>724</x>
      <y>468</y>
      <width>73</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>804</x>
      <y>408</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>angleX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>804</x>
      <y>440</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>angleY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>804</x>
      <y>472</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>angleZ</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtTx">
    <property name="geometry">
     <rect>
      <x>852</x>
      <y>404</y>
      <width>73</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtTy">
    <property name="geometry">
     <rect>
      <x>852</x>
      <y>436</y>
      <width>73</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtTz">
    <property name="geometry">
     <rect>
      <x>852</x>
      <y>468</y>
      <width>73</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>932</x>
      <y>472</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>932</x>
      <y>408</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>932</x>
      <y>440</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ty</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
